libminifi/test/script-tests/ExecuteScriptTests.cpp

```diff
@@ -1,640 +0,0 @@
-/**
```

Comment:
Why were the tests removed? were they not necessary or already covered elsewhere?

Reply:
@phrocker Yes, these tests never should have been in there. The file was a relic from before Python & Lua were split into separate files.